addons/stock/views/stock_quant_views.xml

```diff
@@ -174,15 +190,13 @@
         <field name="code">
             action = model.with_context(
                 search_default_internal_loc=1,
-                search_default_productgroup=1,
-                search_default_locationgroup=1,
+                always_show_loc=1,
             ).action_view_quants()
         </field>
     </record>
 
     <record model="ir.actions.act_window" id="dashboard_open_quants"> <!-- Used in dashboard -->
-        <field name="name">Stock On Hand</field>
-        <field name="context">{'search_default_internal_loc': 1, 'search_default_productgroup':1, 'search_default_locationgroup':1}</field>
+        <field name="name">Locations</field>
```

Comment:
Force an empty value for the context, else the previous one will be kept at the update
```suggestion
        <field name="name">Locations</field>
        <field name="context">{}</field>
```